AirCasting/Graph/Graph.swift

```diff
@@ -73,13 +73,13 @@ class UI_PollutionGraph: UIView {
         lineChartView.leftYAxisRenderer = renderer
     }
     
-    func updateWith(values: [Float]) {
+    func updateWith(thresholdValues: [Float]) {
         guard let renderer = renderer else { return }
-        renderer.thresholds = values
+        renderer.thresholds = thresholdValues
         
         //min & max yaxis values
-        lineChartView.leftAxis.axisMinimum = Double(values.first ?? 0)
-        lineChartView.leftAxis.axisMaximum = Double(values.last ?? 200)
+        lineChartView.leftAxis.axisMinimum = Double(thresholdValues.first ?? 0)
+        lineChartView.leftAxis.axisMaximum = Double(thresholdValues.last ?? 200)
         print("\(lineChartView.leftAxis.axisMinimum) - \(lineChartView.leftAxis.axisMaximum)")
```

Comment:
Please consider using log, and also make the logs clear. This print may make sense for You now but is not understandable out of context where You are not debuging and not analilzing this part of code.

Reply:
Those prints are leftovers from when I was creating the graph, I didn't add them now. 
I guess it's not necessary anymore so I'm removing it.